scripts/importer.py

```diff
@@ -154,6 +159,33 @@ def search_escape(url):
     return url.replace('{', '{{').replace('}', '}}')
 
 
+def opensearch_convert(url):
+    """Convert a basic OpenSearch URL into something Qutebrowser can use.
```

Comment:
Qutebrowser -> qutebrowser (my [pet peeve](https://en.wikipedia.org/wiki/Pet_peeve) :wink:)

(same in "Will obviously trash a properly-formatted Qutebrowser URL." while you're at it)